docs/content/0.1.0/docs/getting-started/docker.md

```diff
@@ -0,0 +1,80 @@
+---
+weight: 10
+---
+
+# Docker
+
+The easiest way to try out DJ is to use the docker setups that are available in
```

Comment:
Can we users to use `dj-demo`? That will start all the necessary services in conjunction (including the jupyter notebook server) so that people can see how it all works together.

Reply:
Ok I updated to open with a `DataJunction Demo Docker Compose Setup` section that specifies that it's the easiest way to try it out. I was going to remove the guide for starting the individual services, but I figured it may actually be useful, even if just to describe how these individual services connect to each other. I also figured it may be useful to describe starting up just the notebook server and updating the DJ server info at the top of the notebook to connect to a remote DJ server.

Reply:
Yeah agreed that it's still useful!